apps/dashboard/src/pages/sdk.tsx

```diff
@@ -135,27 +135,30 @@ const Web3SDK: ThirdwebNextPage = () => {
                 Interact with your contracts from your app in the programming
                 language that you’re familiar with our{" "}
                 <Link
-                  color="blue.500"
+                  className="text-blue-500"
```

Comment:
Please use `text-link-foreground hover:text-foreground` for links

Reply:
updated